### ISSUE TITLE
Please include a readme.txt for the Hexxit 2 Server

### ISSUE THREAD

**Issue by slavinoB** [xJon/Hexxit-II#192]

**Comment by xJon** [comment]
That is only the case if you also have other, incompatible versions of Java installed (e.g. SDK)

**Comment by slavinoB** [comment]
> That is only the case if you also have other, incompatible versions of Java installed (e.g. SDK)

Exactly. If you're a Java Dev like me and my friends, you have jdk and jre installed together and the only information you have is some thrown exception in the cmd. I had to pester the technic support staff for something which can be fixed with a .txt file or an added description to the source website.
